src/mappings.go

```diff
@@ -27,15 +27,18 @@ func (err NothingLinkedError) Error() string {
 type Mappings map[string]abspath.AbsPath
 type MappingsJson map[string]string
 
+var isUnixPlatform = map[string]bool{
+	"linux": true,
+	"darwin": true,
+}
```

Comment:
Please make a simple function like below and use it to check the platform is a unix-like OS. I guess you assumed that more unix-like platform may be added in the future. But I think it's more important to keep code simple for now.

```go
func isUnixLike(p string) bool {
  return p == "linux" || p == "darwin"
}
```